README.md

```diff
@@ -32,44 +33,126 @@ Doing this means:
 We've created this reference guide to provide people who are not developers (and therefore less used to using GitHub) with the essential information they need to be able to **follow the progress of a project**, understand what they're seeing and **participate in the conversation**.
 
 
+## How?
+
+Some sections here are *intended* to be a walkthrough. Feel free to explore Github while *following along* with the guide!
+
+### How to sign up
+
+Go to https://github.com and click 'Sign Up' in the top right! <img width="150" alt="sign-up-box" src="https://cloud.githubusercontent.com/assets/13098199/12653686/8ca8713c-c5e8-11e5-9f9b-e37fa49f7d58.png">
+
+Enter your **username**, **email address** and **password** to create an account.
+
+Then choose the **free tier**, which will be *selected by default* <img width="150" alt="free-tier" src="https://cloud.githubusercontent.com/assets/13098199/12653664/801f1998-c5e8-11e5-8064-70be739a2d74.png">
+
+...and then you'll **officially be a Github user!**
+
+Next, you'll have to **verify your email address** otherwise you will not be allowed to *create* or *comment on* issues (which we'll come to later). You *should* have received an email to **verify your email address**, but if not, do the following:
```

Comment:
Having numbers here might be helpful visually to separate instructions from explanations (like 'what is a repo?')